### DISCUSSION TITLE
windows 环境下 能否调试master、worker呢？

### DISCUSSION THREAD

**Discussion by mrlisir** [apache/dolphinscheduler#7078] [Q&A] [answered]
我的场景是这样的： 
服务器上部署了一整套正常运行的ds。 

但是有时候遇到一些问题，想要windows本地环境下调试master、woker 来调试调度的工作流实例和任务实例。  是否支持 ？ 我查阅了网上相关的所有文章，都找不到答案。故在这里咨询一下。如果有，能否给出相关教程资料？

我在B站上看到的一篇视频：教导如何在windows上运行ds。但仅是运行而已。
https://www.bilibili.com/video/BV1hf4y1b7sX

**Comment by boy-xiaozhang** [comment, marked as answer]
hi running dolphinscheduler from win,tenant encoding validation in the annotation API is required,you can run most scenarios, but you cannot run tasks

**Comment by davidzollo** [comment]
DolphinScheduler can't run in the win environment
